calendar.php

```diff
@@ -596,6 +596,46 @@
 		// Redirect back to the main calendar view.
 		redirect("calendar.php", $lang->redirect_eventdeleted);
 	}
+	else
+	{
+		error($lang->delete_no_checkbox);
```

Comment:
The `headerinclude` template @Destroy666x mentioned below and the new plugin hook.